qoqo_for_braket/qoqo_for_braket/backend.py

```diff
@@ -206,10 +214,22 @@ def _run_circuit(
         Raises:
             ValueError: Circuit contains multiple ways to set the number of measurements
         """
+        (
+            output_bit_register_dict,
+            output_float_register_dict,
+            output_complex_register_dict,
+        ) = self._set_up_registers(circuit)
         (task_specification, shots, readout) = self._prepare_circuit_for_run(circuit)
         return (
             self.__create_device().run(task_specification, shots=shots),
-            {"readout_name": readout},
+            {
+                "readout_name": readout,
+                "output_registers": (
```

Comment:
Is the docstring of this fuction (the return section) still up to date after this change?

Reply:
Updated